README.md

```diff
@@ -8,48 +8,59 @@ https://openslides.com for more information.
 
 __Note: OpenSlides 4 is currently under development.__
 
-### Architecture of OpenSlides 4
 
-![System architecture of OpenSlides 4](docs/OpenSlides4-systemarchitecture.png)
+## Architecture of OpenSlides 4
 
+![System architecture of OpenSlides 4](docs/OpenSlides4-systemarchitecture.png)
 
 Read more about our [concept of OpenSlides 4.0](https://github.com/OpenSlides/OpenSlides/wiki/DE%3A-Konzept-OpenSlides-4).
 
 
 ## Installation
 
-NOTE: Do not use prod at the moment. It will not work. Please refer to the DEVELOPMENT.md!
+### Requirements
 
-Required software: Docker, docker-compose, make, git
+You need [Docker](https://docs.docker.com/engine/install/) and [Docker
+Compose](https://docs.docker.com/compose/install/).
 
-For a non-development setup, clone this repo and run it via docker compose. The make command is a handy shortcut for this:
+### Setup OpenSlides
 
-    $ git clone git@github.com:OpenSlides/OpenSlides.git
-    $ cd OpenSlides
-    $ git checkout openslides4-dev  # needed, until OS4 is released
-    $ make run-prod
+Go to a nice place in your filesystem and get the [OpenSlides manage
+tool](https://github.com/OpenSlides/openslides-manage-service/releases/tag/latest).
 
-For a development setup, refer to [the development docs](DEVELOPMENT.md)
+Create configuration files:
+
+    $ ./manage setup --cwd  # TODO: Provide instruction using XDG_DATA_PATH.
+
+Build and start Docker containers. According to your Docker installation you may
+have to run this as root:
+
+    $ docker-compose up --detach
+
+Setup initial-data:
 
+    $ ./manage initial-data
 
-## Productive environment
+Now you can open http://localhost:8000 and use OpenSlides.
```

Comment:
The setup has to start openslides on https. In other case the browser will use http1.1 that does not perform good with the autoupdate service. You can try this by opening more then four tabs. They will not start the client (since they can not create any http connections. The max connections of your browser to localhost is reached).

Reply:
https://github.com/OpenSlides/OpenSlides/issues/6044